lib/bin/genn-buildmodel.sh

```diff
@@ -64,6 +64,12 @@ fi
 
 # generate model code
 make -f "$GENN_PATH/lib/GNUmakefile" $MACROS
+# The following became necessary because from Mac OSX El Capitan, SIP (System Integrity Protection) squashes DYLD_LIBRARY_PATH in sub-shells and libcudart.8.dylib has a @rpath based install path which necessitates to add the cuda path to DYLD_LIBRARY_PATH.
+unamestr=`uname`
```

Comment:
Did you figure out a better fix for this or is it commented out for another reason?